crates/env/Cargo.toml

```diff
@@ -79,3 +79,13 @@ ink-debug = []
 
 # Disable the ink! provided global memory allocator.
 no-allocator = ["ink_allocator/no-allocator"]
+
+# Configurable sizes of the static buffer
+2GB-buffer = []
+1GB-buffer = []
+512MB-buffer = []
+128MB-buffer = []
+16MB-buffer = []
+1MB-buffer = []
```

Comment:
Starting `1MB` it returns another error:
<img width="636" alt="image" src="https://github.com/paritytech/ink/assets/18346821/91e2df1a-2a6f-43b1-9587-302a67b0b4d6">